tests/unit/core/binarycodec/test_main.py

```diff
@@ -331,14 +331,14 @@ def test_xaddress_xaddr_and_matching_source_tag(self):
 
 
 class TestMainFixtures(TestCase):
-    maxDiff = 1000
+    maxDiff = None
```

Comment:
Why is this field set here if it's not used?

Reply:
It has a default value inherited from `unittest.TestCase` (I forget what it is). This overwrites that value to change it to infinity, basically - so you can see the whole diff (which is pretty necessary when looking at encodings and decodings).